### PR TITLE
Issue10

### PR DESCRIPTION
**Pull request by naz-hage**

### CODE REVIEW

**Comment by ahmdhjj92** [comment]
I like that you created a different file for the results (message, code). It helps to customize later on. One suggestion, can we create worflows in github to run the tests and linters checks? I can help with setting it up.

**Comment by naz-hage** [comment]
I like to learn github actions and workflows.  I created #14 issue to track the work.  
I'll merge this branch once you approve.

**Review by ahmdhjj92 — APPROVED** [review]

**Pull request merged**
